src/gradle-and-npm.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * A `.releaserc.json` cannot specify `/gradle` and `/npm` as separate plugins and run both.
+ * This file allows us to run both behind `/gradleAndNpm`.
+ */
+
+import {
+  createPluginIfFilesExist,
+  createPreset,
+  semanticReleaseGit,
+} from "~/_config";
+
+/**
+ * Semantic release configuration preset for OpenAPI projects. It adds the semantic-release-openapi plugin,
+ * and conditionally the npm and gradle plugins based on the presence of relevant files.
+ */
+export = createPreset([
+  createPluginIfFilesExist(
```

Comment:
Since we now have the `createPluginIfFilesExist` we could probably in a different setup merge all configs into a single one that leverages this to decide what plugins to enable based on what we see in the repo.

It would require some testing, but it would reduce the number of presets we have